README.md

```diff
@@ -35,22 +35,21 @@ Odpri datoteko **taxpayer.xml** in vnesi svoje davčne podatke.
 ### Izvoz poročila v platformi InteractiveBrokers
 
 1. V meniju **Reports** odpri **Flex Queries**
```

Comment:
Meni namesto Reports piše *Performance & Statements* - gledam kaj drugega?

![CleanShot 2022-02-12 at 19 41 25@2x](https://user-images.githubusercontent.com/3621/153724018-2ec8cff1-a97c-4e80-8997-3595db1449b3.png)

Reply:
Ja, sem mislil da je že prej tako pisalo in da se je un Reports nanašal na podfolder. Sem še malo dodatno dopisal.